src/test/java/seedu/progresschecker/logic/parser/ViewTaskListCommandParserTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.progresschecker.logic.parser;
+
+import static seedu.progresschecker.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.progresschecker.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.progresschecker.logic.parser.ParserUtil.MESSAGE_INVALID_TASK_FILTER;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.COMPULSORY;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.COM_INT;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.FIRST_WEEK;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.FIRST_WEEK_INT;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INVALID_CHARSET;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INVALID_NEGATIVE;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.INVALID_ZERO;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.SUBMISSION;
+import static seedu.progresschecker.testutil.TypicalTaskArgs.SUB_INT;
+
+import org.junit.Test;
+
+import seedu.progresschecker.logic.commands.ViewTaskListCommand;
+
+//@@author EdwardKSG
+public class ViewTaskListCommandParserTest {
+
+    private ViewTaskListCommandParser parser = new ViewTaskListCommandParser();
+
+    @Test
+    public void parse_validArgs1_returnsViewTaskListCommand() {
```

Comment:
Here too. :) Maybe names like ```parse_validWeekNumber_returnViewTaskListCommand``` would make what the method is testing for clearer?